gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/aggregation/functions/SetAggregateProperty.java

```diff
@@ -63,6 +68,7 @@ public SetAggregateProperty(Set<AggregateFunction> aggregateFunctions) {
     for (AggregateFunction func : aggregateFunctions) {
       checkNotNull(func);
     }
+    this.aggregateFunctions = aggregateFunctions;
```

Comment:
Does not depend on your PR, but please put the null check of the first loop in the second loop and place your assignment below.